src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -2175,3 +2142,44 @@ export function getEditorSelectionFromMatch(element: FileMatchOrMatch, viewModel
 	}
 	return undefined;
 }
+
+export function getSelectionTextFromEditor(allowUnselectedWord: boolean, editor: IEditor, configurationService: IConfigurationService): string | null {
```

Comment:
I'm not seeing `configurationService`  used in this function... is that right?

Reply:
Oh yes, after some iterations he no longer needs this service.